package.json

```
@@ -27,6 +27,9 @@
     "build:image": "npm run build && docker buildx build --output type=docker --tag pepr:dev .",
     "test": "npm run test:unit && npm run test:journey",
     "test:unit": "npm run gen-data-json && jest src --coverage --detectOpenHandles --coverageDirectory=./coverage --testPathIgnorePatterns='cosign.e2e.test.ts'",
+    "test:int": "npm run test:int:prep && npm run test:int:run",
+    "test:int:prep": "./integration/prep.sh",
+    "test:int:run": "jest --maxWorkers=4 integration",
```

```suggestion
    "test:integration": "./integration/prep.sh && jest --maxWorkers=4 integration",
```

Alternatively, based on the PR we really only use `test:int` in CI and the nested commands of `prep` and `run` only contain one command each.

So, in interest of avoiding a situation where `A` calls `B` & `C` where `B` calls `1` and `C` calls `2` we can have `A` call `1` and `2` directly. Less bloat for an already chunky `scripts` section.

---

Nah, would rather not collapse these for dev-flow purposes.

The benefit of the separation is that I can just run the "test:integration:prep" step once (to get the .tgz installed) and then I can run the "test:integration:run" command a _bunch_ of times as I iterate on the tests.